rust/ethers-prometheus/src/middleware/mod.rs

```diff
@@ -627,71 +539,14 @@ impl<M: Middleware + Send + Sync> PrometheusMiddleware<M> {
         }
         if let Some(gas_price_gwei) = gas_price_gwei {
             if let Some(london_fee) = current_block.base_fee_per_gas {
-                let gas = u256_as_scaled_f64(london_fee, 18) * 1e9;
+                let gas = u256_as_scaled_f64(london_fee.into(), 18) * 1e9;
```

Comment:
curious why the `into()` is now needed, iiuc no types have changed?

Reply:
oops, leftover from me playing around